tests/PHPStan/Levels/data/arrayDimFetches.php

```diff
@@ -55,7 +55,7 @@ public function doBaz($a, $b): void
 	 */
 	public function iterableOffset($iterable): void
 	{
-		unset($iterable['foo']);
+		var_dump($iterable['foo']);
```

Comment:
Leftover Debug output?

Reply:
No, I just wanted to change this line to some kind of function call that doesn’t cause `isUndefinedExpressionAllowed === true` (that means something except `isset, unset, empty`)

Reply:
Please add new code instead of changing existing tests :) Thanks.